libs/s25main/ingameWindows/iwSave.cpp

```diff
@@ -142,24 +143,33 @@ iwSave::iwSave() : iwSaveLoad(40, _("Save game!"))
     /// Combobox füllen
     combo->AddString(_("Disabled")); // deaktiviert
 
-    // Last entry is only for debugging
-    const unsigned numIntervalls = SETTINGS.global.debugMode ? NUM_AUTO_SAVE_INTERVALS : NUM_AUTO_SAVE_INTERVALS - 1;
-
     // Die Intervalle
-    for(unsigned i = 0; i < numIntervalls; ++i)
-        combo->AddString(helpers::toString(AUTO_SAVE_INTERVALS[i]) + " GF");
+    for(unsigned i = 0; i < NUM_AUTO_SAVE_INTERVALS; ++i)
+        combo->AddString((boost::format(_("%1% m")) % AUTO_SAVE_INTERVALS[i]).str());
+
+    // Last entry is only for debugging
+    if(SETTINGS.global.debugMode)
+    {
+        combo->AddString(_("Every GF"));
+    }
 
     // Richtigen Eintrag auswählen
     bool found = false;
-    for(unsigned i = 0; i < numIntervalls; ++i)
+    constexpr unsigned GFsPerMinute = 60 * 1000 / SPEED_GF_LENGTHS[referenceSpeed];
+    for(unsigned i = 0; i < NUM_AUTO_SAVE_INTERVALS; ++i)
     {
-        if(SETTINGS.interface.autosave_interval == AUTO_SAVE_INTERVALS[i])
+        if(SETTINGS.interface.autosave_interval == AUTO_SAVE_INTERVALS[i] * GFsPerMinute)
```

Comment:
This can then be: 
```suggestion
        if(SETTINGS.interface.autosave_interval == AUTO_SAVE_INTERVALS[i] / std::chrono::milliseconds(SPEED_GF_LENGTHS[referenceSpeed])
```
Or introduce a constant `referenceGFLength` or so.
Reason is a) easier to read and b) avoid manual time computation via magic numbers

Reply:
makes sense